app/src/main/res/layout-land/profile_edit_fragment.xml

```diff
@@ -33,7 +33,7 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-      <de.hdodenhof.circleimageview.CircleImageView
+      <com.google.android.material.imageview.ShapeableImageView
```

Comment:
Continuing the discussion from https://github.com/oppia/oppia-android/pull/4139#discussion_r796231929.

I see that not all uses of ShapeableImageView specify an appearance overlay, but more do since my last review. Should all of them?

Reply:
Initially I didn't added appearance overlay property to all the files but now I do think it should be added to all.
What are your thoughts on it? 
Should I add this property to all files?

Reply:
Do the other views look correct? I'm actually a bit confused why they all look correct if only some have the property defined (since your image above suggests not defining this property results in a square image). Can you provide some context on why the views not using the property look correct (or if they don't)?

Reply:
Done

Reply:
I have added the appearance overlay property for other views which were missing and now all are circular.